apps/projects/app/components/Card/Issue.js

```diff
@@ -45,6 +56,52 @@ FlexibleDiv.propTypes = {
   children: PropTypes.node.isRequired,
 }
 
+const Bounty = ({ issue }) => {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const { workStatus, deadline, expLevel, openSubmission } = issue
+
+  let status = BOUNTY_STATUS[workStatus]
+  if (openSubmission && workStatus === 'funded') status = 'Accepting work submissions'
+
+  return (
+    <React.Fragment>
+      <FlexibleDiv compact={layoutName !== 'large'}>
+        <span css="white-space: nowrap">
+          <IconAddUser
+            color={`${theme.surfaceIcon}`}
+            css="margin-bottom: -8px; margin-right: 4px"
+          />
+          {status}
+        </span>
+        {dot}
+        <span css="white-space: nowrap">
+          <div css={`
+              display: inline-block;
+              vertical-align: bottom;
+              margin-right: 6px;
+              margin-bottom: -8px;
+            `}>
```

Comment:
Why does `IconClock` require a special `div` wrapper, but the other icons do not?

Reply:
Apparently it doesn't. It was there before, but the relevant bit is only that -8px margin. I made it more standard (that component's local standard).